x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/blocklist_validator.ts

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENDPOINT_BLOCKLISTS_LIST_ID } from '@kbn/securitysolution-list-constants';
+import { schema, TypeOf } from '@kbn/config-schema';
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { OperatingSystem } from '@kbn/securitysolution-utils';
+import { BaseValidator } from './base_validator';
+import { ExceptionItemLikeOptions } from '../types';
+import {
+  CreateExceptionListItemOptions,
+  UpdateExceptionListItemOptions,
+} from '../../../../../lists/server';
+import { isValidHash } from '../../../../common/endpoint/service/trusted_apps/validations';
+import { EndpointArtifactExceptionValidationError } from './errors';
+
+const ProcessHashField = schema.oneOf([
+  schema.literal('process.hash.md5'),
```

Comment:
Check with @joeypoon . I think these are actually incorrect for Blocklist - I think they are `file.hash.**`. He was making changes yesterday around this.